handler/oauth2/flow_authorize_code_auth.go

```diff
@@ -77,6 +77,9 @@ func (c *AuthorizeExplicitGrantHandler) HandleAuthorizeEndpointRequest(ctx conte
 		return nil
 	}
 
+	if ar.GetResponseMode() == fosite.ResponseModeNone {
+		ar.SetResponseMode(fosite.ResponseModeQuery)
```

Comment:
I think response mode spec does not influence what is being returned, only how. And if anything of what is being returned is sensitive, then how it can be returned is restricted.